app/concepts/lockstep/api_record.rb

```diff
@@ -572,13 +572,6 @@ def self.create(attributes = {})
       obj
     end
 
-    # Replaced with a batch destroy_all method.
```

Comment:
Should we remove the commented code?

Reply:
I have replaced the same method right? So what is the issue with removing the commented one? ITs anyway just running destroy in a loop

Reply:
No what I mean is not keeping this commented in the code, since it's already replaced by a new code.

Reply:
bro red mean deleted only no xD?